tests/integration/sdk/test_vault.py

```
@@ -34,17 +34,15 @@ async def fulfill_github_login(route):  # pyright: ignore[reportUnknownParameter
-        password="xyz",
-    )
-    with notte.Session() as session:
-        agent = notte.Agent(session=session, vault=vault, max_steps=5)
-        _ = agent.run(task="Go to the github.com and try to login with the credentials")
-
-    _ = client.vaults.delete(vault.vault_id)
+    with client.Vault() as vault:
+        _ = vault.add_credentials(
+            url="https://github.com/",
+            email="xyz@notte.cc",
+            password="xyz",
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "password"

(S106)